bigquery/snippets/loadingdata/bigquery_add_column_load_append.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
```

Comment:
Probably don't need bigquery in any of the file names. Might also be able to shorten based on the package/directory this is in (may not need `load`).

Reply:
To prevent myself going absolutely bonkers, the samples are named for the sample tag since they're now one sample per file.